packages/forms/backend/sirius-components-widget-reference/src/main/resources/schema/reference.graphqls

```diff
@@ -1,3 +1,7 @@
+extend type EditingContext {
+  referenceValueOptions(representationId: ID!, referenceWidgetId: ID!): [ReferenceValue!]!
```

Comment:
Do not place your fields like that directly on EditingContext because everyone will start using this global object as the go to place to put everything. Place that on the description like the palette of the diagram, this way it stays within the domain of the form representation.

It would also remove the need to ask for a representation id while there is already an API to go this way. Look at how the diagram tools are retrieved in the GraphQL API.

Reply:
Done